lgc/builder/DescBuilder.cpp

```diff
@@ -39,6 +40,14 @@
 using namespace lgc;
 using namespace llvm;
 
+namespace {
+const unsigned DescriptorSizeResource = 8 * sizeof(unsigned);
```

Comment:
I think this info should be a part of gpu property

Reply:
OK, but I think I should not put the compact buffer and ycbcr sampler sizes in there as they are driver constructs and nothing to do with hardware.